math/mathcore/src/TMath.cxx

```diff
@@ -440,6 +440,24 @@ Double_t TMath::BreitWigner(Double_t x, Double_t mean, Double_t gamma)
    return bw/(2*Pi());
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// Calculate a Relativistic Breit Wigner function with mean and gamma.
```

Comment:
Can you add in the documentation the mathematical formula of the function in latex format. 
See example at https://github.com/root-project/root/blob/master/math/mathcore/inc/Math/SpecFuncMathCore.h#L43